src/main/java/mateacademy/internetshop/web/New.java

```diff
@@ -0,0 +1,4 @@
+package mateacademy.internetshop.web.filters;
+
+public class New {
```

Comment:
do we need this class?

Reply:
no, it is here only because web and filter folders shown as one folder without it...
I remove it